_locales/en.xml

```diff
@@ -755,6 +755,8 @@
     <string name="pref_edit_profile">Edit Profile</string>
     <string name="disable_imap_idle">Disable IMAP IDLE</string>
     <string name="disable_imap_idle_explain">Do not use IMAP IDLE extension even if the server supports it. Enabling this option will delay message retrieval, enable it only for testing.</string>
+    <string name="enable_realtime">Enable Realtime APIs</string>
+    <string name="enable_realtime_explain"> Enable Webxdc realtime APIs which create direct connection between devices and leak IP addresses to each other.</string>
```

Comment:
these belong not here but in `_locales/_untranslated_en.json` instead, or in a pr to the android repo. this file is not to be edited by developers as it is overwritten with the next pull from transifex.